lunar_lander/main.py

```diff
@@ -2,11 +2,43 @@
 import time
 
 env = gym.make("LunarLander-v2")
-actions = {"nop": 0, "fire_left": 1, "main_engine": 2, "right_engine": 3}
+actions = {"no_action": 0, "left_engine": 1, "main_engine": 2, "right_engine": 3}
 
 
+def heuristic_tyro(observation, t):
+    x_pos, y_pos, x_vel, y_vel, angle, angular_vel, leg1_contact, leg2_contact = observation
+    obs_dict = {
+        "x_pos": x_pos,
+        "y_pos": y_pos,
+        "x_vel:": x_vel,
+        "y_vel": y_vel,
+        "angle": angle,
+        "angular_vel": angular_vel,
+        "leg1_contact": leg1_contact,
+        "leg2_contact": leg2_contact
+    }
+    print(obs_dict)
+    action = None
+    if obs_dict["leg1_contact"] == 1 or leg2_contact == 1:
```

Comment:
🔧 Maybe like this for consistency?
```suggestion
    if leg1_contact == 1 or leg2_contact == 1:
```

Reply:
Good suggestion 👍